pkgs/desktops/plasma-5/plasma-workspace/default.nix

```
@@ -46,6 +46,18 @@ mkDerivation {
   patches = [
     ./0001-startkde.patch
     ./0002-absolute-wallpaper-install-dir.patch
+    (# Systemd v246 hides the interface used by KDE to fetch the
+     # user session details.
+     # We're applying this commit from 5.19.90 upgrading the
+     # systementry applet to use the new sessionmanagement API.
+     # See https://github.com/NixOS/nixpkgs/issues/98141 for more
+     # details.
+     # /!\ Remove this patch for version >= v5.19.90
+      fetchurl {
```

The indentation here is a little unconventional for Nixpkgs, but the important part is to use `fetchpatch`:

```suggestion
    # Systemd v246 hides the interface used by KDE to fetch the
    # user session details.
    # We're applying this commit from 5.19.90 upgrading the
    # systementry applet to use the new sessionmanagement API.
    # See https://github.com/NixOS/nixpkgs/issues/98141 for more
    # details.
    # /!\ Remove this patch for version >= v5.19.90
    (fetchpatch {
```

---

ack regarding fetchpatch.

The comment is encapsulated in the `(` to make sure it'll go away with the patch.